boards/arm/efm32gg_slwstk6121a/efm32gg_slwstk6121a.dts

```diff
@@ -114,7 +114,7 @@
 };
 
 &eth0 {
-	local-mac-address = [02 01 02 03 04 05];
+	/* local-mac-address = <>;*/
```

Comment:
This mirrors the definition from stk3701a. If it's better to leave it out entirely, I'm happy to remove it from both definitions.

Reply:
I think it would be better to remove this line, after all one should never hard code the MAC so that all the devices would get the same MAC address.